.github/workflows/tests-with-odoo.yml

```
@@ -8,11 +8,12 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-${{ matrix.ubuntu_version }}
     strategy:
       fail-fast: false
       matrix:
         odoo_release: ["12.0", "13.0", "14.0", "15.0"]
+        ubuntu_version: ["bionic", "focal", "latest"]
```

Maybe the nicknames are not recognized. See https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources. And that document indicates that latest is a synonym for 20.04, not 22.04 :confused: 
```suggestion
        ubuntu_version: ["18.04", "20.04", "22.04"]
```